### ISSUE TITLE
Unable to generate logs: can't compare offset-naive and offset-aware datetimes

### ISSUE THREAD

**Issue by JCDetwiler** [ProtonVPN/linux-cli#94]
The CLI client has the following error for me on Arch Linux:

```
$ protonvpn-cli --get-logs

Generating logs...

Unable to generate logs: can't compare offset-naive and offset-aware datetimes
```

**Comment by Tnology** [comment]
Hey, did you ever fix this issue?

**Comment by JCDetwiler** [comment]
> Hey, did you ever fix this issue?

No, sorry. It's a standing issue with the repo because Proton abandoned their official client...again. There haven't been any commits here in several months.

**Comment by S0litaire** [comment]
Started getting this today out of the blue!. 
To my knowledge, nothing on my system has changed

**Comment by cyberfeign** [comment]
I'm getting this as well. Trying to troubleshoot "Unknown API error." whenever I try to log in with either CLI or GUI.
